pkg/appgateserver/metrics.go

```diff
@@ -0,0 +1,36 @@
+package appgateserver
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	// relaysTotal is a counter metric that represents the total number of requests processed by the AppGate server.
+	// It is a Counter type metric, which means it is used to accumulate counts of events (in this case, the total number of requests).
+	// This metric is crucial for understanding the overall workload and traffic handled by the server.
+	// It is labeled by 'service_id', which allows for distinguishing and aggregating request counts across different services managed by the AppGate server.
+	//
+	// Example of usage:
+	// - Monitoring the aggregate load and request rate over time.
+	// - Comparing request volumes across different services managed by the AppGate server.
+	relaysTotal metrics.Counter = prometheus.NewCounterFrom(stdprometheus.CounterOpts{
```

Comment:
Wdyt of adding a `Counter` postfix to every metric. It'll make ctrl-shift-f easier.

Open to pushback.

Reply:
@Olshansk though there are no standards, "_total" is the advised suffix for a counter per Prometheus naming style guide: https://prometheus.io/docs/practices/naming/.

<img width="664" alt="Screenshot 2024-01-25 at 1 08 18 PM" src="https://github.com/pokt-network/poktroll/assets/4950477/6a0111d1-2380-4618-b0bf-be5288e45f8e">

Reply:
SGTM!